rune/libenclave/internal/runtime/pal/skeleton/Makefile

```diff
@@ -80,16 +63,14 @@ EXTRA_CLEAN := \
 	$(OUTPUT)/encl.elf \
 	$(OUTPUT)/encl.ss \
 	$(OUTPUT)/sgx_call.o \
+	$(OUTPUT)/aesm.o \
+	$(OUTPUT)/aesm.pb-c.* \
```

Comment:
$(OUTPUT)/aesm.pb-c.c is not good enough?

Reply:
protoc-c generate two files: aesm.pb-c.c and aesm.pb-c.h